containers/kubernetes/how-to/upgrade-kubernetes-version.mdx

```diff
@@ -1,22 +1,27 @@
 ---
 meta:
-  title: How to upgrade your Kubernetes Kapsule cluster
-  description: This page explains how to upgrade your Kubernetes Kapsule cluster
+  title: How to upgrade the Kubernetes version on a Kapsule cluster
+  description: This page explains how to upgrade the Kubernetes version on a Kapsule cluster
 content:
-  h1: How to upgrade your Kubernetes Kapsule cluster
-  paragraph: This page explains how to upgrade your Kubernetes Kapsule cluster
-tags: kubernetes kubernetes-kapsule kapsule cluster 
+  h1: How to upgrade the Kubernetes version on a Kapsule cluster
+  paragraph: This page explains how to upgrade the Kubernetes version on a Kapsule cluster
+tags: kubernetes kubernetes-kapsule kapsule cluster upgrade
 dates:
-  validation: 2022-12-21
+  validation: 2023-06-01
   posted: 2022-12-21
 categories:
   - kubernetes
 ---
 
 <Macro id="iam-requirements" />
 
+
+You can [apply the latest patches](#applying-patches-to-your-cluster-from-the-scaleway-console) to your Kubernetes cluster directly from the Scaleway console. If you want to [upgrade your cluster to the next minor release](#upgrading-to-the-next-minor-version-using-the-cli), you can achieve this using the CLI.
```

Comment:
The console can also upgrade to the next minor release, not just latest patches. 
But the CLI section of this tutorial however provides very helpful tips for checking important components before proceeding.